loglist2/logfilter.go

```diff
@@ -16,24 +16,25 @@ package loglist2
 
 import (
 	"github.com/golang/glog"
+	"github.com/google/certificate-transparency-go/trillian/ctfe"
 	"github.com/google/certificate-transparency-go/x509"
 )
 
-// LogRoots maps Log-URLs (stated at LogList) to the list of their accepted
+// LogRoots maps Log-URLs (stated at LogList) to the pools of their accepted
 // root-certificates.
-type LogRoots map[string][]*x509.Certificate
+type LogRoots map[string]*ctfe.PEMCertPool
 
-// SelectQualified creates a new LogList containing only qualified logs from
+// SelectUsable creates a new LogList containing only usable logs from
 // the original.
-func (ll *LogList) SelectQualified() LogList {
+func (ll *LogList) SelectUsable() LogList {
 	var active LogList
 	active.Operators = make(map[string]*Operator)
 	// Keep all the operators but filter Logs.
 	for opName, op := range ll.Operators {
-		active.Operators[opName] = &Operator{Email: make([]string, len(op.Email)), Logs: make(map[string]*Log)}
-		copy(active.Operators[opName].Email, op.Email)
+		active.Operators[opName] = &Operator{Logs: make(map[string]*Log)}
+		active.Operators[opName].Email = op.Email
```

Comment:
nit: nicer to fold these 2 lines together
```golang
	active.Operators[opName] = &Operator{Email: op.Email, Logs: make(map[string]*Log)}
```

Reply:
Done